### PR TITLE
Multiarch: revert the image mirroring logic for different architectures

### PR DESCRIPTION
**Pull request by danilo-gemoli**
This PR is related to [DPTP-3351](https://issues.redhat.com/browse/DPTP-3351)

/cc @droslean

### CODE REVIEW

File: hack/image-mirroring/mirror-images.sh
Comment (droslean) [inline hack/image-mirroring/mirror-images.sh:1]: We don't need to remove this script. I would prefer keeping it and changing all image mirroring jobs to use the script instead of inlining a bash code into the args.

File: ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml
Comment (droslean) [inline ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml:957]: This needs to be removed in another PR in advanced.

Comment (danilo-gemoli) [reply ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml:957]: Ok so we should open another PR, remove this presubmit and then come back and keep working on this.

Comment (droslean) [reply ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml:957]: we should merge this PR right after the presubmit removal.

**Comment by danilo-gemoli** [comment]
/hold
we need to have #37288 merged

**Comment by danilo-gemoli** [comment]
/retitle Multiarch: revert the image mirroring logic for different architectures

File: hack/image-mirroring/supplemental_ci_images_mirror_gen.py
Comment (droslean) [inline hack/image-mirroring/supplemental_ci_images_mirror_gen.py:27]: You need to remove this file as well.

Comment (danilo-gemoli) [reply hack/image-mirroring/supplemental_ci_images_mirror_gen.py:27]: Deleted in [b958cbd5b320ae3d55b8fa5723e499127f3669d5](https://github.com/openshift/release/pull/37274/commits/b958cbd5b320ae3d55b8fa5723e499127f3669d5)
```bash
$ git diff upstream/master b958cbd5b32 --summary -- hack/image-mirroring/validate-multi-arch-image-mirror-gen.sh
 delete mode 100755 hack/image-mirroring/validate-multi-arch-image-mirror-gen.sh
```

Comment (droslean) [reply hack/image-mirroring/supplemental_ci_images_mirror_gen.py:27]: I mean the file that is in the comment. `clusters/app.ci/multi-arch/namespaces.yaml`

**Review by droslean — APPROVED** [review]

**Pull request merged**
